Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -112,6 +113,9 @@ @interface OWSProfileManager ()
 // This property should only be mutated on the main thread,
 @property (nonatomic, readonly) NSCache<NSString *, UIImage *> *otherUsersProfileAvatarImageCache;
 
+// This property should only be mutated on the main thread,
+@property (atomic, readonly) NSMutableSet<NSString *> *currentAvatarDownloads;
```

Comment:
We use this to ensure that only one download is active at a time for a given profile avatar.